requirements/rpms.txt

```diff
@@ -1,6 +1,7 @@
 # To compile C bindings in certain Python dependencies
 gcc-8.5.0
 git-2.31.1
+golang-bin-1.17.12-1.module+el8.6.0+16014+a372c00b
```

Comment:
Just curious, is this installing the `-bin` package specifically for a reason?

Reply:
It's a smaller package and include the `go` package which we need. See https://developers.redhat.com/blog/2017/03/17/basics-of-go-in-fedora#how_to_install_fedora_golang_

That's what deplist installs as well.

Reply:
I think both `dnf install golang` and `dnf install golang-bin` results in the same package set. `golang-bin` is just a subpackage of `golang`, which it needs anyway.